docs/5g-on-helium/5g-poc.mdx

```diff
@@ -119,20 +117,23 @@ must:
   - Be registered with the SAS (Outdoor Radios Only).
 - Broadcast at least 1 "heartbeat" per Reward Period.
 
-### Genesis Phase 3: Speed Test and Quality of Service
+### Genesis Phase 3: Speed Tests and Quality of Service
 
+> - Reward Period is approximately 1440 blocks and closely aligns with 00:00 UTC to 23:59 UTC.
+> - Reward submission occurs approx 15 minutes after the Reward Period ends, around 00:00 UTC.
 > - 5G Hotspots perform periodic speed tests which will be considered in reward calculations.
 > - A minimum Download speed, Upload speed, and Latency will be required by each 5G Hotspot to
 >   ensure it is providing quality coverage.
 
-A 5G Hotspot will perform a speed test every 12 hours with the results being averaged over the
-previous 6 tests.
-
 To meet the speed test requirements as outlined in
 [HIP-53](https://github.com/helium/HIP/blob/main/0053-mobile-dao.md#oracle-operations), a 5G Hotspot
 must have a rolling average Download speed of at least 100Mbps **AND** a rolling average Upload
 speed of at least 10Mbps **AND** a rolling average Latency of 50ms or less.
 
+A 5G Hotspot will perform a speed test approximately every 12 hours with the results being averaged
```

Comment:
This might lead people to think that speed tests are averaged on the hotspot. It is worth mentioning here how re-tries work.